integration-cli/requirements.go

```diff
@@ -32,6 +33,13 @@ var (
 		func() bool { return supportsExec },
 		"Test requires 'docker exec' capabilities on the tested daemon.",
 	}
+	Network = TestRequirement{
+		func() bool {
+			_, err := http.Get("http://google.com")
```

Comment:
@tianon then we can count on the number of interfaces being `>0` if loopback is always going to be there.